### PR TITLE
Copilot chat cleanup - removing Vite and updating README

### PR DESCRIPTION
**Pull request by teresaqhoang**
### Motivation and Context

Removing Vite dependency to simplify toolchain + updating README

### Description

We are authenticating with the /common endpoint, and in order to use /common/ endpoint, the application must not be configured with 'Consumer' as the user audience. The userAudience should be configured with 'All' to use /common/ endpoint.

### Contribution Checklist

- [X] The code builds clean without any errors or warnings
- [X] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [X] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [X] All unit tests pass, and I have added new tests where possible
- [X] I didn't break anyone :smile:

### CODE REVIEW

File: samples/apps/copilot-chat-app/WebApp/src/components/views/BackendProbe.tsx
Comment (craigomatic) [inline samples/apps/copilot-chat-app/WebApp/src/components/views/BackendProbe.tsx:1]: Given you are making changes in this file, would you mind also updating the path we are telling the developer to look for the backend service? It should be something like `samples/apps/copilot-chat-app/SKWebApi`

Comment (teresaqhoang) [reply samples/apps/copilot-chat-app/WebApp/src/components/views/BackendProbe.tsx:1]: Done!

**Comment by teresaqhoang** [comment]
@adrianwyatt waiting on for a feature description for Shannon to update README (but non-blocking, we can update it in a later PR)

File: samples/apps/copilot-chat-app/WebApp/README.md [hunk 1]
```diff
@@ -7,15 +13,20 @@
...
    in the Azure Portal. Follow the steps to register your app
    [here](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app).
-    - Select **`Single-page application (SPA)`** as platform type, and the Redirect URI will be **`https://localhost:3000`**
-    - Select **`Personal Microsoft accounts only`** as supported account types for this sample
+    - Select **`Single-page application (SPA)`** as platform type, and the Redirect URI will be **`http://localhost:3000`**
+    - Select **`Accounts in any organizational directory and personal Microsoft accounts`** as supported account types for this sample.
 4. Create an **[.env](.env)** file to this folder root with the following variables and fill in with your information, where
-   `REACT_APP_GRAPH_CLIENT_ID=` is the GUID copied from the **Application (client) ID** from the Azure Portal and
-   `VITE_REACT_APP_BACKEND_URI=` is the URI where your backend is running:
+   `REACT_APP_CHAT_CLIENT_ID=` is the GUID copied from the **Application (client) ID** from the Azure Portal and
```
Comment (hathind-ms) [inline samples/apps/copilot-chat-app/WebApp/README.md:19]: I don't think we need to mention REACT in the name.

**Review by adrianwyatt — APPROVED** [review]

**Pull request merged**